src/content-scripts/pageTransition.event.content.js

```diff
@@ -128,8 +128,8 @@
          * property from this module's click content script and background script. The cached page
          * visits are from the same tab as this page or, if the page is opened in a new tab, the cached
          * page visits are from the opener tab.
-         * @param {boolean} message.isOpenedTab - Whether the page is loading in a new tab that was
-         * opened by another tab.
+         * @property {number} message.openerTabId - If the page is loading in a tab that was newly
+         * opened from another tab, the tab ID of the opening tab. Otherwise -1.
```

Comment:
```suggestion
         * @property {number} message.openerTabId - If the page is loading in a tab that was newly
         * opened from another tab (i.e., `isOpenedTab` is `true`), the tab ID of the opener tab. Otherwise, `tabs.TAB_ID_NONE`.
```